Prepare web worker for AWS

muscar opened the pull request:
This PR adds a `/health` endpoint that does the same thing as `/health/ping` to make ALB health checks work. It also adds a toggle for disabling HTTPS which is not supported between ALBs and EC2 instances.

Commit:
Prepare web worker for AWS

This PR adds a `/health` endpoint that does the same thing as
`/health/ping` to make ALB health checks work. It also adds a toggle for
disabling HTTPS which is not supported between ALBs and EC2 instances.

tompave-roo commented on routemaster/application.rb:
```diff
@@ -18,7 +17,11 @@ class Application < Sinatra::Base
 
-    use Rack::SSL
+    if ENV.fetch('FORCE_SSL', 'true') =~ /YES|TRUE|ON|1/i
+      require 'rack/ssl'
```
Ok, but if you want to lazily require it, then you can also add `require: false` in the gemfile.

Commit:
Update circle config to work with ECS specific branches

To keep the branch history clean, we're going to use dedicated ECS
branches (prefixed with `ecs_`) for new infra related commits. This PR
updates the circle config to take this into account.
